File: 7KYU/Vowel Count/solution.rs
fn get_count(string: &str) -> usize {
    let mut vowels_count: usize = 0;
    for c in string.chars() { 
        if "aeiou".contains(c) {
            vowels_count=vowels_count + 1;
        }
    }
  
    vowels_count
}

File: 7KYU/Vowel Count/solution.js
function isVowel(x) {
    if (x == 'a' || x == 'e' || x == 'i' || x == 'o' || x == 'u') {
      return true;
    }
    return false;
}
  
function getCount(str) {
    var cnt = 0;
    
    for(var i = 0; i < str.length; i++) {
      if(isVowel(str[i]))
        cnt++;
    }
    return cnt;
}